doc/contributing/primordials.md

```diff
@@ -122,6 +122,8 @@ performance of code in Node.js.
 * `SafePromiseAny`
 * `SafePromiseRace`
 * `SafePromisePrototypeFinally`: use `try {} finally {}` block instead.
+* `ReflectConstruct`: see linked PRs to [nodejs/performance#109](https://github.com/nodejs/performance/issues/109)
```

Comment:
```suggestion
* `ReflectConstruct`: Also effected `Reflect.construct`. `ReflectConstruct` creates new types of classes inside functions. Instead create a shared class. See [nodejs/performance#109](https://github.com/nodejs/performance/issues/109)
```

Reply:
Thanks for your suggestion but I think I will create a new documentation file just to talk about the performance issues and better alternatives to cover cases where the primordial is not slow, just the operation itself (like Reflect.Construct).

Reply:
For the record, either options (Aras' suggestion, or Vinicius' plan) sounds good to me, and would resolve my objection.

Reply:
In this case, if the suggestion is good enough, I'd rather keep it simple and just add this suggestion for now.

And later I can create a doc file and add more context about these alternatives (instead of just making contributors see a bunch of PRs).